src/frontend/src/pages/FinancePage/ReimbursementRequestDetailPage/SubmitToSaboModal.tsx

```diff
@@ -129,11 +129,11 @@ const SubmitToSaboModal = ({ open, setOpen, reimbursementRequest }: SubmitToSabo
           <Stack>
             <Box display="flex" alignItems="center">
               <Typography>Alex Leblang</Typography>
-              <CopyToClipboardButton msg={'Brody Pearlman'} />
+              <CopyToClipboardButton msg={'Alex Leblang'} />
```

Comment:
Lets tie the display and copy values to a variable so they both get updated no matter what next time

Reply:
yeah sounds good